Server/Classes/Handlers/ServerEventHandlers.sqx

```diff
@@ -76,7 +76,7 @@ namespace Intrusion.Server
 		{
 			_unit addEventHandler ["GetInMan", {
 				params ["_unit" as Object, "_role" as String, "_vehicle" as Object, "_turret" as Array];
-				["ClientEventReciever.OnUnitEnteredVehicle", [_unit, [_role] call VehicleRoleMeta.ToVehicleRole, _vehicle, _turret]] call Remote.Invoke;
```

Comment:
Personally I think we should check if role is "TURRET" and in that case convert it to "GUNNER". Alternatively we should rename the VehicleRole enum to VehiclePosition. Just to keep the integrity in the semantics.